local/kind/setup.sh

```diff
@@ -65,6 +65,7 @@ for cluster in "${CLUSTERS[@]}"; do
     clusterExists=""
     if echo "${EXISTING_CLUSTERS}" | grep "$cluster"; then
         clusterExists="1"
+        ${KIND_CMD} export kubeconfig --name "$cluster" --kubeconfig "${TMP_DIR}/${cluster}.kubeconfig"
```

Comment:
fixes the issue when the argoCD install fails in case kind clusters were already running